packages/protoc-gen-es/README.md

```diff
@@ -31,45 +24,92 @@ pete = Person.fromBinary(bytes);
 pete = Person.fromJsonString('{"name": "pete", "id": 123}');
 ```
 
-Learn more at [github.com/bufbuild/protobuf-es](https://github.com/bufbuild/protobuf-es).
+Learn more about the project at [github.com/bufbuild/protobuf-es](https://github.com/bufbuild/protobuf-es).
 
 
 ## Installation
 
-### With npm
+`protoc-gen-es` is a code generator plugin for Protocol Buffer compilers,
+like [buf](https://github.com/bufbuild/buf) and [protoc](https://github.com/protocolbuffers/protobuf/releases).
+It generates base types - messages and enumerations - from your Protocol Buffer 
+schema. The generated requires the runtime library [@bufbuild/protobuf](https://www.npmjs.com/package/@bufbuild/protobuf).
+
+To install the plugin and the runtime library, run:
 
 ```shell
-npm install @bufbuild/protoc-gen-es
+npm install --save-dev @bufbuild/protoc-gen-es
+npm install @bufbuild/protobuf
 ```
 
-This will install the code generator plugin in `node_modules/.bin/protoc-gen-es`. 
-Note that npm does not add the executable to your `$PATH`. You can do so with:
+We use peer dependencies to ensure that code generator and runtime library are
+compatible with each other. Note that yarn and pnpm only emit a warning in this case.
```

Comment:
Hoping this might help a bit with the situation regarding https://github.com/bufbuild/connect-web/issues/239.